roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+# Roles and Groups
+
+This file contains an overview of our rules for roles and groups.
+
+## Definition of general roles
+
+The Hiero project (and the LFDT) defines 3 different types of **roles**: **Contributors**, **Committers**, and **Maintainers**.
+Each person who contributes to Hiero must follow our [**Code of Conduct**](https://github.com/hiero-ledger/.github/blob/main/CODE_OF_CONDUCT.md).
+Those roles are not assigned globally for the Hiero project but per **project**.
+In most cases a project is a single GitHub repository, but it can also be a group of repositories that are related to each other.
+A person can have multiple roles in multiple projects.
+
+### Contributors
+
+Any person who contributes to a project is a Contributor. A contribution can be anything from opening an issue, to writing code, to writing documentation, etc.
+
+  - If a pull request has not received an approval within 2 business days or a discussion has no activity for more than 2 business days, the owner must reach out to the maintainers via our [hiero-general Discord channel](https://discord.lfdecentralizedtrust.org/).
+  - If a pull request or discussion has gone stale for more than 2 weeks, the repo maintainers may close them due to inactivity.
+
+We honor legitimate contributions to the project, and we want to make sure that all contributions are recognized.
+
+A Contributor can be nominated by a Committer or a Maintainer to become a Committer:
+
+  - **Start Contributing**: Submit pull requests, report bugs, improve documentation, review code. Show consistency and quality in your contributions.
```

Comment:
The list that starts here does not fit to the introduction.

Reply:
I think is fitting, since these are pointers for new "Contributors" in specific. I think it makes sense to have them in the "Contributors" section. Otherwise, there is no section where these guidelines can go. 

This list is basically guidelines for Contributors in order to build history and get nominated by a Maintainer or Committer.